CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
+## 1.5.1
```

Comment:
Yeah, I waffled a bit on it too.  I guess arguably it's a "new feature" and so 'y' may make more sense.  But I tend to think of it as just exposing a new way to configure an already existing feature so not sure the 'y' is totally warranted.

In the end, though, I really don't have a strong opinion about this either.